### DISCUSSION TITLE
KubernetesPodOperator cannot list resource in API group in the namespace

### DISCUSSION THREAD

**Discussion by ghost** [apache/airflow#21856] [General]
### Apache Airflow version

2.2.3

### What happened

I used the KubernetesPodOperator in another namespace that airflow server is using and I've got an rbac rejection:

`cannot list resource in API group in the namespace`

Is it possible to add rbac rights to one of the airflow serviceaccount ?

### Operating System

Ubuntu

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by potiuk** [comment]
This is at most a discussion no issue. @rdeteix  - please avoid creating issues when you are explicitly warned during Issue creation that you should open discussion in such cases.
